PyTorch/resnet50/README.md

```diff
@@ -260,4 +260,4 @@ Done! with highest_accuracy:  0
 - [Alternative training data (CIFAR-10)](https://www.cs.toronto.edu/~kriz/cifar.html)
 
 Alternative implementations:
-- [ONNX](https://github.com/onnx/models/tree/master/vision/classification/resnet)
\ No newline at end of file
+- [ONNX](https://github.com/onnx/models/tree/master/vision/classification/resnet)
```

Comment:
ORT has since renamed their branch from "master" to "main". How about:

https://github.com/onnx/models/tree/main/vision/classification/resnet

Reply:
done

Reply:
Cool. I added Xiang and Sheil because they actually work on PyTorch (I work on DirectML), but since this is just a documentation edit, I'll approve. Thanks for the fixes.